jpo-geojsonconverter/src/main/java/us/dot/its/jpo/geojsonconverter/converter/FieldConversions.java

```diff
@@ -295,25 +364,213 @@ public record SecondNanos(Integer secondOfMinute, Integer nanoOfSecond) {
      * @return Java ZoneOffset
      */
     public static ZoneOffset convertDOffset(DOffset dOffset) {
-        if (dOffset == null) return ZoneOffset.UTC;
+        if (dOffset == null)
+            return ZoneOffset.UTC;
         final int value = (int) dOffset.getValue();
         int offsetHours = Math.floorDiv(value, 60);
         int offsetMinutes = value - (offsetHours * 60);
         return ZoneOffset.ofHoursMinutes(offsetHours, offsetMinutes);
     }
 
+    /**
+     * Calculates the lane width based on the lane width meter and lane width offset in centimeters.
+     *
+     * @param laneWidthMeter J2735 lane width meter value.
+     * @param laneWidthOffsetCm J2735 lane width offset in centimeters.
+     * @return Lane width in meters.
+     */
+    public static Double calculateLaneWidthOffset(Double laneWidthMeter, long laneWidthOffsetCm) {
+        Double returnValue = null;
+        returnValue = laneWidthMeter + (laneWidthOffsetCm * 1e-2);
+        return returnValue;
+    }
+
+    /**
+     * Calculates the elevation based on the elevation meter and elevation offset in centimeters.
+     *
+     * @param elevationMeter J2735 elevation meter value.
+     * @param elevationOffsetCm J2735 elevation offset in centimeters.
+     * @return Elevation in meters.
+     */
+    public static Double calculateElevationOffset(Double elevationMeter, long elevationOffsetCm) {
+        Double returnValue = null;
+        if (elevationMeter != null) {
+            returnValue = elevationMeter + (elevationOffsetCm * 1e-2);
+        }
+        return returnValue;
+    }
+
+    /**
+     * Parse heading sectors directly from Asn1Bitstring. Each bit represents a 22.5-degree sector starting from North
+     * (0°) and moving clockwise.
+     *
+     * @param directionBitstring The Asn1Bitstring direction field
+     * @return Array of active sector bit positions
+     */
+    public static int[] parseHeadingSectorsFromBitstring(Asn1Bitstring directionBitstring) {
```

Comment:
issue: this method is not used
suggestion: looks like it was intended to be used by the method below it